src/App.js

```diff
@@ -1,25 +1,25 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import Home from './components.js/Home';
+// import NavBar from './components.js/NavBar';
```

Comment:
- [ ] kindly remove commented code for cleaner code practice

Reply:
well resolved! ✔️